lte/gateway/deploy/roles/magma/files/systemd/sctpd.service

```diff
@@ -11,14 +11,15 @@
 #
 [Unit]
 Description=Magma sctpd service
-PartOf=magma@mme.service
 Before=magma@mme.service
 
 [Service]
 Type=simple
 EnvironmentFile=/etc/environment
 ExecStartPre=-/bin/cp -f /usr/local/share/sctpd/version /var/run/sctpd.version
+ExecStartPre=/usr/local/bin/config_stateless_agw.sh sctpd_pre
 ExecStart=/usr/local/sbin/sctpd
+ExecStartPost=/usr/local/bin/config_stateless_agw.sh sctpd_post
```

Comment:
is there a cycle created in stateless mode, i.e., sctpd restart -> mme restart -> sctpd restart ?

Reply:
In stateless mode, mme restart will not trigger sctpd exit, as it can reuse the existing SCPT connection. MME terminates sctpd only in stateful mode, so that SCTP socket is forced to be released and then re-opened.